crypto/crypto.go

```diff
@@ -178,6 +178,9 @@ func UnmarshalPubkey(pub []byte) (*ecdsa.PublicKey, error) {
 	if x == nil {
 		return nil, errInvalidPubkey
 	}
+	if !S256().IsOnCurve(x, y) {
```

Comment:
can you explain this a little bit? do we need Tarak to look at this at all?

Reply:
The relevant CVE is this one: https://github.com/advisories/GHSA-q26p-9cq4-7fc2 .
However, this seems to only affect Geth nodes, and we do not use the `go-ethereum` dependency for that purpose, neither in `flow-go` nor in `flow-evm-gateway`.
So the `UnmarshalPubkey` function isn't really touched at all in the repositories where we use the `go-ethereum` dependency.